docs/api.md

```diff
@@ -1118,19 +1118,19 @@ This endpoint supports retrieving extended metadata; provide a header `X-Extende
 
 _(introduced: version 2022.3)_
 
-Dataset is a collection of entities. Dataset schema is created using form XML, you can see [ODK XForm](https://getodk.github.io/xforms-spec) specification for defining dataset in form XML. 
+A Dataset is a collection of Entities. A Dataset's schema is defined by the Forms that create Entities in that Dataset. See the [ODK XForms specification](https://getodk.github.io/xforms-spec) for defining Datasets in Form definitions.
 
-Entities are added to dataset when a submission of the form that defines the  dataset is **approved**.
+Entities are added to a Dataset when a Submission of a Form that creates Entities is **approved**.
 
 ### Related APIs:
```

Comment:
I wish there was a way to move this heading below; after all the actions of current resource group